### PR TITLE
fix: remove libcfbuildpack

### PR DESCRIPTION
**Pull request by elbandito**
Remove dependency on `libcfbuildpack`

### CODE REVIEW

File: function/buildpack.go [hunk 1]
```diff
@@ -56,9 +56,11 @@ func Detect(bp Buildpack) {
 }
 
 func doDetect(bp Buildpack, d detect.Detect) (int, error) {
+	// Look for metadata toml file that includes artifact, handler, and override fields that the
+	// user can overwrite.  We should considering using this for Heroku or just removing it.
```
Comment (murphybytes) [inline function/buildpack.go]: I'm not sure what the last sentence in this comment means.

File: function/metadata.go [hunk 2]
```diff
@@ -23,15 +23,13 @@ import (
...
-	"github.com/cloudfoundry/libcfbuildpack/logger"
+	"github.com/buildpack/libbuildpack/logger"
 )
 
 const (
-	RiffEnv     = "RIFF"
-	ArtifactEnv = "RIFF_ARTIFACT"
-	HandlerEnv  = "RIFF_HANDLER"
-	OverrideEnv = "RIFF_OVERRIDE"
+	ArtifactEnv = "ARTIFACT"
```
Comment (murphybytes) [inline function/metadata.go]: If these constants are package public they should have descriptive comments.  Do they need to be public?

File: function/metadata.go [hunk 3]
```diff
@@ -23,15 +23,13 @@ import (
...
+	HandlerEnv  = "HANDLER"
+	OverrideEnv = "OVERRIDE"
 )
 
 // Metadata represents the contents of the riff.toml file in an application root
```
Comment (murphybytes) [inline function/metadata.go]: Is the name of riff.toml in the comment correct?

File: function/metadata.go [hunk 4]
```diff
@@ -87,3 +85,16 @@ func NewMetadata(application application.Application, logger logger.Logger) (Met
 	logger.Debug("riff metadata: %s", metadata)
 	return metadata, true, nil
 }
+
+func fileExists(file string) (bool, error) {
```
Comment (murphybytes) [inline function/metadata.go]: Is this included in your test coverage? If it's not you should write a test.

**Review by murphybytes — CHANGES_REQUESTED** [review]

File: function/metadata.go (hunk 4 above)
Comment (elbandito) [reply function/metadata.go]: yeah, it's being covered.

**Review by murphybytes — APPROVED** [review]

**Pull request merged**